### ISSUE TITLE
Converter results wrong for array schemas with no explicit collectionFormat

### ISSUE THREAD

**Issue by brhaible** [Mermade/oas-kit#189]
The conversion result is wrong when the OpenAPIv2 input contains a schema of type "array" with no explicit 'collectionFormat'.

### Checklist

- [X] Conversion: I have checked my source definition is valid OpenAPI 2.0
- [X] Conversion: On valid OpenAPI 2.0 input, the conversion looks wrong

### Detailed Description

The OpenAPIv2 spec https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#parameterObject is clear that, for parameters of type "array", the default value of 'collectionFormat' is "csv".

However, swagger2openapi converts it as if the default was "multi".

For an example, see the attached files.
How to reproduce:
```
$ swagger2openapi -o query-parameter-array-without-collectionFormat.v3.yaml query-parameter-array-without-collectionFormat.v2.yaml
```

In the v3 output, we have

          schema:
            type: array
            items:
              type: string

The correct output would be

          schema:
            type: array
            style: form
            explode: false
            items:
              type: string

Why? Because per OpenAPIv3 spec https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md
1) The default value for 'style' in query parameters is "form".
2) The default value for 'explode' is true in this case.
3) The combination of 'style: form' and 'explode: true' corresponds to 'collectionFormat: multi'.

To get the equivalent of 'collectionFormat: csv', you need to add
```
            explode: false
```
or
```
            style: form
            explode: false
```

#### Other stuff
Version: swagger2openapi-5.3.0

[query-parameter-array-without-collectionFormat.v2.yaml.gz](https://github.com/Mermade/oas-kit/files/3442500/query-parameter-array-without-collectionFormat.v2.yaml.gz)
[query-parameter-array-without-collectionFormat.v3.yaml.gz](https://github.com/Mermade/oas-kit/files/3442501/query-parameter-array-without-collectionFormat.v3.yaml.gz)

**Comment by brhaible** [comment]
This patch fixes  it for me.

[issue189-fix.diff.gz](https://github.com/Mermade/oas-kit/files/3442543/issue189-fix.diff.gz)

**Comment by MikeRalphson** [comment]
Apologies for the delay, should be fixed in:
 - swagger2openapi@5.3.2

**Comment by brhaible** [comment]
Thanks. I confirm it's fixed in swagger2openapi@5.3.2.
